tasklist/client/src/v2/Tasks/AvailableTasks/index.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+
+import {useEffect, useRef} from 'react';
+import {useTranslation} from 'react-i18next';
+import {Stack} from '@carbon/react';
+import {Search} from '@carbon/react/icons';
+import {useTaskFilters} from 'v1/features/tasks/filters/useTaskFilters';
+import type {Task as TaskType} from 'v1/api/types';
```

Comment:
Do we want to use v2 types here?

Reply:
The change is done on the following PR, this was just to reduce the diff